packages/run-it/src/components/ResponseExplorer/ResponseExplorer.spec.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2021 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+import React from 'react'
+import { renderWithTheme } from '@looker/components-test-utils'
+import { screen, waitFor } from '@testing-library/react'
+import { IRawResponse } from '@looker/sdk-rtl'
+import userEvent from '@testing-library/user-event'
+import { ResponseExplorer, ResponseHeaders } from '.'
+
+const response: IRawResponse = {
+  url: '/sample/url',
+  ok: true,
+  statusCode: 200,
+  statusMessage: 'OK',
+  body: Buffer.from('plain text'),
+  contentType: 'text/plain',
+  headers: {
+    'content-type': 'text/plain',
+    link: 'nah, not really',
+    'x-total-count': '0',
+  },
+}
+
+describe('ResponseExplorer', () => {
+  test('ShowHeaders', async () => {
+    renderWithTheme(<ResponseHeaders response={response} />)
+    const heading = 'Headers (3)'
+    const header = screen.getByText(heading)
+    expect(header).toBeInTheDocument()
+    await userEvent.click(header)
+    await waitFor(() => {
+      // TODO figure out a more specific selector?
+      const table = screen.getByRole('table')
```

Comment:
I assume you're saying this because there are (or there could be) multiple tables in the page. In that case you could use [`within`](https://testing-library.com/docs/dom-testing-library/api-within/).

Although it doesn't apply to this particular component, it's worth mentioning that there are places where it would also help to use the `data-testid` prop ([reference](https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change#whats-with-the-data-testid-query)). This would also make end to end tests easier.